src/App/App.test.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import App from './App.js';
+import { App } from './App.js';
 import { container, render, screen, waitFor, querySelector } from '@testing-library/react';
```

Comment:
I think we should make this a convention, just doing the `export`  and wrapping the imports in curlies.